assets/src/phoenix/app.ts

```diff
@@ -17,7 +17,18 @@ import { initActivityBridge } from './activity_bridge';
 const csrfToken
   = (document as any).querySelector('meta[name="csrf-token"]').getAttribute('content');
 const liveSocket
-  = new LiveSocket('/live', Socket, { hooks: Hooks, params: { _csrf_token: csrfToken } });
+  = new LiveSocket('/live', Socket, {
+    hooks: Hooks,
+    params: { _csrf_token: csrfToken },
+    metadata: {
+      keydown: (e: any, el: any) => {
+        return {
+          key: e.key,
+          shiftKey: e.shiftKey,
```

Comment:
This is necessary to trap the "shiftKey" pressed modifier from the javascript event and pass it to liveview actions